.github/workflows/antsibull-build.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           python3 -m pip install poetry ansible-core
           poetry install
           poetry update
+          ansible-galaxy collection install community.general
```

Comment:
No, please don't do that (vendoring).

But instead of using `ansible-galaxy collection install`, which is (due to Ansible Galaxy) not very reliable, let's use `git clone` instead :-) This should work:
```suggestion
          git clone --depth=1 --single-branch https://github.com/ansible-collections/community.general.git collections/ansible_collections/community/general
```

Reply:
What about `ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.general.git'`?

Reply:
> What about `ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.general.git'`?

I've added this and force pushed. This also has the benefit of installing the latest released version instead of directly from `main`.

Reply:
> What about `ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.general.git'`?

I think that does basically the same as just cloning the repo, except that it also builds a collection artifact out of it, and then installs that collection. I.e. same result, just slower.

> This also has the benefit of installing the latest released version instead of directly from `main`.

Are you sure? I just tried `ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.network.git'`, and it did install from `main` and not from the `4.0.0` tag.

Reply:
``` console
$ rpm -q ansible-core
ansible-core-2.13.1-1.fc35.noarch
$ ansible-galaxy collection install 'git+https://github.com/ansible-collections/community.general.git' -p /tmp                                                 
Cloning into '/home/gotmax/.ansible/tmp/ansible-local-1447931_ovbb3g9/tmpgt_zqrc0/community.general61wqlyji'...
remote: Enumerating objects: 2728, done.
remote: Counting objects: 100% (2728/2728), done.
remote: Compressing objects: 100% (2039/2039), done.
remote: Total 2728 (delta 328), reused 1615 (delta 275), pack-reused 0
Receiving objects: 100% (2728/2728), 2.64 MiB | 1.87 MiB/s, done.
Resolving deltas: 100% (328/328), done.
Your branch is up to date with 'origin/main'.
Starting galaxy collection install process
[WARNING]: The specified collections path '/tmp' is not part of the configured Ansible collections paths '/home/gotmax/.ansible/collections:/usr/share/ansible/collections'. The installed collection won't be picked up in an Ansible run.
Process install dependency map
Starting collection install process
Installing 'community.general:5.3.0' to '/tmp/ansible_collections/community/general'
Created collection for community.general:5.3.0 at /tmp/ansible_collections/community/general
community.general:5.3.0 was installed successfull
```

Reply:
You're right. It's still installing from main and the version there is presumably just picked up from galaxy.yml. 5.3.0 hasn't been released yet and it explicitly says `Your branch is up to date with 'origin/main'.` /me did not read the output closely enough when I ran it the first time. Let me know if you still want me to change it to just manually git clone.